src/api/promptEnhancer.ts

```diff
@@ -0,0 +1,41 @@
+import { MessageContent } from 'shared/model';
+import { Api } from './api'
+import { parseError } from './error'
+
+async function enhancePrompt(messageContent: MessageContent): Promise<MessageContent> {
+  try {
+    if (!messageContent.attachment) {
+      return messageContent;
+    }
+
+    let image;
+    if (messageContent.attachment.image) {
+      image = messageContent.attachment.image;
+    }
+    else if (messageContent.attachment.figma) {
+      if (!messageContent.attachment.figma.imageUrl) {
+        // TODO: Warning no image url
```

Comment:
LGTM just add warnings here. You can use `console.warn` until we add some logger later